react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -443,29 +768,58 @@ class Thumbnail extends Component<Props, State> {
     _renderRemoteParticipant() {
         const {
             _audioTrack,
+            _isTestModeEnabled,
             _participant,
-            _startSilent
+            _startSilent,
+            _videoTrack
         } = this.props;
         const { id } = _participant;
-        const { audioLevel, volume } = this.state;
+        const { audioLevel, canPlayEventReceived, volume } = this.state;
+        const styles = this._getStyles();
+        const containerClassName = this._getContainerClassName();
 
         // hide volume when in silent mode
         const onVolumeChange = _startSilent ? undefined : this._onVolumeChange;
-        const jitsiTrack = _audioTrack?.jitsiTrack;
-        const audioTrackId = jitsiTrack && jitsiTrack.getId();
+        const jitsiAudioTrack = _audioTrack?.jitsiTrack;
+        const audioTrackId = jitsiAudioTrack && jitsiAudioTrack.getId();
+        const jitsiVideoTrack = _videoTrack?.jitsiTrack;
+        const videoTrackId = jitsiVideoTrack && jitsiVideoTrack.getId();
+        const videoEventListeners = {};
+
+        if (_videoTrack && _isTestModeEnabled) {
+            Object.keys(VIDEO_TEST_EVENTS).forEach(attribute => {
```

Comment:
for..in ?

Reply:
It turns out I even don't need a Map here... I suppose this is leftovers from previous version of the refactoring...

Will use array instead!